.github/lineage.yml

```diff
@@ -0,0 +1,5 @@
+---
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Comment:
Shouldn't this role descend from `skeleton-ansible-role`?
```suggestion
    remote-url: https://github.com/cisagov/skeleton-ansible-role.git
```

Reply:
Great catch!